Berechnung in CoPE.md

```diff
@@ -242,10 +242,11 @@ In dieser Schleife werden die Werte für den Vorhersagezeitraum berechnet
 Before we can use the loop we need to generate data for the past so that the `GesamtAktuellInfizierteBerechnet` can be calculated
 
 ```r
-  offsetDay <- ceiling(log(n0_erfasst*faktor_n_inf,Rt)) # calculate the day when one case was there 
-  calcInit <- data.frame(Tag = seq(startDate- offsetDay, startDate, by = 1))  %>% 
+ Rt_start <- Rt
+ offsetDay <- ceiling(log(n0_erfasst*faktor_n_inf,Rt)) # calculate the day when one case was there 
+  calcInit <- data.frame(Tag = seq(startDate- offsetDay, startDate, by = 1)) %>% 
     mutate(indexBack = as.numeric(-(Tag - startDate)),
-           TaeglichReproduktionsRateRt       = Rt,
+           TaeglichReproduktionsRateRt       = Rt_start + (Rt_start-1)*indexBack/Y_inf_limit,
```

Comment:
change doesn't effect result but is a better way to calculate data prior to startDate